### PR TITLE
-Implemented tickets #22, #23, #25, and #26

### PR DESCRIPTION
**Pull request by Platnumrcn14**
-Rules, logic, and game flow for Blackjack coded
-Populates deck at beginning of game, cleans deck after each game
-if statements in PlayBlackjack class to be populated by buttons in future

### CODE REVIEW

**Review by eBerkley — APPROVED** [review]
I'll add a couple minor changes, this can be merged though.

File: CardsCashCasino/BlackjackManager.cs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                    case Value.Seven:
+                    case Value.Eight:
+                    case Value.Nine:
+                    case Value.Ten:
+                        value += (int)card.Value; // Number cards are worth their face value
+                        break;
+                    case Value.Jack:
+                    case Value.Queen:
+                    case Value.King:
+                        value += 10; // Face cards are worth 10 points each
```
Comment (modmo22) [inline CardsCashCasino/BlackjackManager.cs:67]: There's a way to get the BlackjackValue of these cards from that attribute of the face card Enum objects that we should use. I'll approve this PR if Ethan includes this in his minor edits of this code.

File: CardsCashCasino/BlackjackManager.cs [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                        break;
+                    case Value.Ace:
+                        aceCount++; // Track the number of aces
+                        break;
+                }
+            }
+
+            // Add ace values, either as 11 or 1, as appropriate
+            for (int i = 0; i < aceCount; i++)
```
Comment (modmo22) [inline CardsCashCasino/BlackjackManager.cs:76]: Same here. We can use the attribute of the Ace enum value to implement this logic.

File: CardsCashCasino/BlackjackManager.cs [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+                // Draw a new card for each hand
+                _playerHand.Add(_cardManager.DrawCard());
+                _playerSplitHand.Add(_cardManager.DrawCard());
+            }
+        }
+        /// <summary>
+        /// Determines if the player has won, lost, or if the game is still ongoing.
+        /// </summary>
+        public string GetGameStatus(List<Card> hand)
```
Comment (modmo22) [inline CardsCashCasino/BlackjackManager.cs:107]: You'll have to make sure to set the _gameOver flag to True for each else if block

**Review by modmo22 — APPROVED** [review]

**Pull request merged**